packages/app/src/app/pages/Dashboard/index.js

```diff
@@ -22,89 +22,85 @@ import {
   OffsettedLogo,
 } from './elements';
 
-class Dashboard extends React.Component {
-  state = {
-    showSidebar: false,
-  };
+const Dashboard = props => {
+  const [showSidebar, setShowSidebar] = useState(false);
 
-  componentDidMount() {
-    this.props.signals.dashboard.dashboardMounted();
-  }
+  const {
+    state: { hasLogIn },
+    actions: { dashboard },
+  } = useOvermind();
 
-  componentWillUnmount() {
-    // Reset store so new visits get fresh data
-    client.resetStore();
-  }
+  useEffect(() => {
+    dashboard.dashboardMounted();
```

Comment:
How to fix this type error ? 🤔
![image](https://user-images.githubusercontent.com/3725386/67034657-8515ca00-f131-11e9-913c-45e4faf46fb0.png)

Reply:
All CI checks (including typecheck) are green so It seems that this typecheck error was my environment specific.

Reply:
May be an old thing that already got fixed

Reply:
CI typecheck is now failing because of this error after renaming the file to tsx.

Reply:
Fixed